src/Repository/ConferenceRepository.php

```diff
@@ -63,6 +63,17 @@ public function findOneAttended(string $slug): ?Conference
         ;
     }
 
+    public function findOneByHash(string $hash): ?Conference
+    {
+        $qb = $this->createQueryBuilder('c')
+            ->where('c.hash = :hash')
+            ->setParameter('hash', $hash)
+            ->getQuery()
+            ->getOneOrNullResult();
+
+        return $qb;
+    }
```

Comment:
Doctrine has `findOneBy` method, which is allow you to find one entity by the first argument. i.e: `$repo->findOneBy('hash', $hash)`. So, you could remove this method